### PR TITLE
Russell Roberts HW1 (Resubmission)

### PR DESCRIPTION
**Pull request by russell-roberts**
Following the tutorial on Github, editing the Rmd file was fairly straightforward. Looking forward to getting further into the course content this week!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,21 @@
-# hw01
\ No newline at end of file
+# Homework 1
+
+## Rusty Roberts
+
+My name is Rusty! I was born in Cleveland, Ohio, the *best* city in the world! You might agree, you might not, but you can't argue with the *emphasis* I've placed on the word *best* to make my claim more compelling. After attending college at the College of Wooster in Wooster, Ohio, I worked as a teachcer in New York City for eight years. I'm now a doctoral student at Booth studying behavioral science.
```
Comment (elliotkarl) [inline README.md:5]: I wasn't convinced at first...but the emphasis changed my mind!

File: README.md [hunk 2]
```diff
@@ -1 +1,21 @@
...
+My name is Rusty! I was born in Cleveland, Ohio, the *best* city in the world! You might agree, you might not, but you can't argue with the *emphasis* I've placed on the word *best* to make my claim more compelling. After attending college at the College of Wooster in Wooster, Ohio, I worked as a teachcer in New York City for eight years. I'm now a doctoral student at Booth studying behavioral science. 
+
+### Favorites!
+
+**Foods**
+* Spaghetti
+* Tacos
+* Popcorn
+
+**TV Shows**
```
Comment (elliotkarl) [inline README.md:14]: Nice use of lists; you can tab to indent the sub-items and make it more readable

**Review by elliotkarl — COMMENTED** [review]
Solid assignment -- good use of links, lists, emphasis.

File: README.md [hunk 3]
```diff
@@ -1 +1,21 @@
...
+* Star Trek
+* The Wire
+* Breaking Bad
+
+### Links!
+
+[Cute Dogs](https://www.goodhousekeeping.com/life/pets/g4531/cutest-dog-breeds/)
```
Comment (rrodriguezbarron) [inline README.md:21]: These dogs are **extremely** cute...

**Review by rrodriguezbarron — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |         x  |              |            |
| **Coding strategy**         |          |     x         |            |
| **Presentation**            |           |     x         |            |
| **Achievement, creativity** |      x     |              |            |
| **Ease of access**          |           |     x         |            |

Remarks:

* Great assignment
* The lists and emphasis were correctly coded
* The link to the webpage worked
* You should also consider practicing how to load images from both URLs and files.

**Pull request closed without merging**